vcd/resource_vcd_vapp.go

```diff
@@ -78,6 +78,10 @@ func resourceVcdVApp() *schema.Resource {
 			"metadata": {
 				Type:     schema.TypeMap,
 				Optional: true,
+				/*
```

Comment:
Nit: please use `//` comment in this case, which would be more compact (two lines as opposed to four without a need to indent the text).

Reply:
Fixed.